SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialReplicationGraph.cpp

```diff
@@ -15,9 +19,19 @@ void USpatialReplicationGraph::InitForNetDriver(UNetDriver* InNetDriver)
 	{
 		if (USpatialNetDriver* SpatialNetDriver = Cast<USpatialNetDriver>(InNetDriver))
 		{
-			LoadBalancingHandler = MakeUnique<FSpatialLoadBalancingHandler>(SpatialNetDriver);
+			bStrategyWorkerEnabled = USpatialStatics::IsStrategyWorkerEnabled();
+			if (bStrategyWorkerEnabled && SpatialNetDriver->LoadBalanceStrategy)
+			{
+				bDirectAssignment = SpatialNetDriver->LoadBalanceStrategy->IsStrategyWorkerAware();
+			}
+			if (!bStrategyWorkerEnabled)
+			{
+				LoadBalancingHandler = MakeUnique<FSpatialLoadBalancingHandler>(SpatialNetDriver);
```

Comment:
Should `bDirectAssignment` be false in this case?

Reply:
Removed bDirectAssignment as wasn't needed